client/src/components/Home/PartnersSection/PartnersSection.css

```diff
@@ -125,14 +125,14 @@ text-decoration: none;
       }
 }
 
-@media(min-width:1600px) {
+@media(min-width:1400px) {
       .container.partners-section {
             max-width: 85vw;
       }
 
       .partners-logo, .logo-name {
             width: 25vw;
-            border-radius: 35px;
+            border-radius: 30px;
       }
 
       .logo-name {

```

Comment:
On hovering over the IEEE logo the ```logo-name``` is touching the edges which don't look good. I would suggest you can increase the width to 23vw or any other value which fits the best.

Reply:
@hariom1625 changed the width to 23vw